src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -175,6 +175,30 @@ namespace AppInstaller::Manifest
                 }
             }
 
+            if (installer.InstallerType == InstallerTypeEnum::Zip)

```

Comment:
Would be better as:
```suggestion
            if (IsArchiveType(installer.InstallerType))
```
to make the intention more clear for the future.

Reply:
Changed to suggested